README.md

```diff
@@ -120,7 +120,53 @@ FLAGS = {
 
 The set of conditions can be none (flag will never be enabled), one (only condition that has to be met for the flag to be enabled), or many (all have to be met for the flag to be enabled).
 
-Additional conditions can be added in the Django or Wagtail admin for any defined flag (illustrated in [Usage](#usage)). Conditions added in the Django or Wagtail admin can be changed without restarting Django, conditions defined in `settings.py` cannot.
+Additional conditions can be added in the Django or Wagtail admin for any defined flag (illustrated in [Usage](#usage)). Conditions added in the Django or Wagtail admin can be changed without restarting Django, conditions defined in `settings.py` cannot. See below [for a list of built-in conditions](#built-in-conditions).
+
+### Using flags in code
+
+Flags can be used in Python code:
+
+```python
+from flags.state import flag_enabled
+
+if flag_enabled('MY_FLAG', request=a_request):
+	print("My feature flag is enabled")	
+```
+
+Django templates:
+
+```django
+{% flag_enabled 'MY_FLAG' as my_flag %}
```

Comment:
It might be nice to include the

```html
{% load feature_flags %}
```

line here (as above in Quickstart) as this is necessary for this tag to work.